### DISCUSSION TITLE
Weapon holder cannot be interacted after put anything inside

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14406] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Only have this issue in the editor

### Reproduction steps

1 place a weapon holder
2 put some thing in
3 close it
4 press E

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
You mean in sub editor right? It seems to be because the "auto-equip" behavior is overwriting the selection input, the same problem happens with fire extinguisher holders. 

Strangely however, diving suit lockers are not affected by this bug for some reason despite being "auto-equipped"
(and this bug didn't happen in previous versions)

For now a workaround seems to be to right-click the container and press "open"

**Reply by jcx1991911** [reply]
Every update bring some of new "surprises", at least it have a temporary solution for now )

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/4f886ed6c36bfab49734fd7cc0262f4e50a9d996 (commit in our private repo, fix coming up in the next update). Sorry for the inconvenience!
